ckanext/unaids/assets/webassets.yml

```diff
@@ -64,3 +64,14 @@ MapSelectorCSS:
     filters: cssmin
     output: unaids/%(version)s_MapSelector.css
 
+main_css:
```

Comment:
I think we need to rename things here? Maybe ValidationBadgeCSS and ValidationBadgeJS?